detections/endpoint/windows_obfuscated_files_or_information_via_rar_sfx.yml

```
@@ -0,0 +1,68 @@
+name: Windows Obfuscated Files or Information via RAR SFX
```

The logic is looking for the creation of that file. So the title needs to reflect that.

I didn't do full testing but from what i could gather online is that this file will get created every time an SFX is executed hence the title should be something like `Windows SFX Temporary File Creation` or something along these lines.

```suggestion
name: Windows Obfuscated Files or Information via RAR SFX
```

---

thank you man for the review. great point, currently the format of naming convention we have is <OS><Mitre Technique/Tactic><Name>. also we have limited file name length I think :)

---

Its noted. We'll discuss a bit more internally.